src/components/panel/panel.js

```diff
@@ -200,6 +201,34 @@ angular
  * @returns {!MdPanelAnimation} panelAnimation
  */
 
+/**
+ * @ngdoc method
+ * @name $mdPanel#newPanelGroup
+ * @description
+ * Creates a panel group and adds it to a tracked list of panel groups.
+ *
+ * @param groupName {string} Name of the group to create.
+ * @param config {!Object=} Specific configuration object that may contain the
+ *     following properties:
+ *
+ *   - `maxOpen` - `{number=}`: The maximum number of panels that are allowed
+ *     open within a defined panel group.
+ *
+ * @returns {!Object} group
```

Comment:
I think you need a type for `Object` here. I have no idea what "group" is going to contain or how to use it when I get it back. Do I have access to my config? Are there special functions I can call?